README.md

```diff
@@ -161,21 +161,7 @@ following rules:
     * FLOAT, DOUBLE - Signed comparison with special handling of NaNs and
       signed zeros.   The details are documented in the
       [Thrift definition](src/main/thrift/parquet.thrift) in the
-      `ColumnOrder` union. They are summarized here but the Thrift definition
```

Comment:
So this part is removed and unified into the `parquet.thrift`?

Reply:
Indeed, as was [requested in this issue](https://github.com/apache/parquet-format/pull/196#discussion_r1151335207). I do agree that not duplicating it is better.